import-export-cli/impl/mg/deleteAPI.go

```diff
@@ -0,0 +1,43 @@
+/*
+*  Copyright (c) WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
+*
+*  WSO2 Inc. licenses this file to you under the Apache License,
+*  Version 2.0 (the "License"); you may not use this file except
+*  in compliance with the License.
+*  You may obtain a copy of the License at
+*
+*    http://www.apache.org/licenses/LICENSE-2.0
+*
+* Unless required by applicable law or agreed to in writing,
+* software distributed under the License is distributed on an
+* "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+* KIND, either express or implied.  See the License for the
+* specific language governing permissions and limitations
+* under the License.
+ */
+
+package mg
+
+import (
+	"errors"
+	"net/http"
+
+	"github.com/wso2/product-apim-tooling/import-export-cli/utils"
+)
+
+// DeleteAPI sends a POST request to delet an API
+func DeleteAPI(accessToken string, apiDeleteEndpoint string, queryParam map[string]string) (
```

Comment:
```suggestion
func DeleteAPI(accessToken, apiDeleteEndpoint string, queryParam map[string]string) (
```

Reply:
We can takeout the common string type as shown above in go. Shall we change like that?

Reply:
Thanks. Will remove the first "string".